src/proof/lrat/lrat_proof.cpp

```diff
@@ -239,7 +311,7 @@ LratProof::LratProof(std::istream& textualProof)
       {
         Assert(textualProof.good());
         // Create an entry in the RAT hint list
-        resolvants.emplace_back(make_pair(-i, std::vector<ClauseIdx>()));
+        resolvants.emplace_back(i, std::vector<ClauseIdx>());
```

Comment:
It looks like you changed from `-i` to `i`. Was that on purpose?

Reply:
Good catch, thanks!